org.kde.kongress.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "org.kde.kongress",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kongress",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.Solid.PowerManagement",
+        "--own-name=org.kde.kongressac"
+    ],
+    "modules": [
+        {
+            "name": "kcalendarcore",
```

Comment:
you should not need to build calendarcore, it's part of the sdk

Reply:
Done